onnx/defs/printer.cc

```diff
@@ -247,7 +247,25 @@ void ProtoPrinter::print(const TensorProto& tensor, bool is_initializer) {
         printSet(" {", ",", "}", ParseData<double>(&tensor));
         break;
       default:
-        output_ << "..."; // ParseData not instantiated for other types.
+        // For unsupported data types, show raw bytes as hex
+        if (tensor.has_raw_data()) {
+          const std::string& raw = tensor.raw_data();
+          output_ << " {0x";
+          // Save current format state
+          auto saved_flags = output_.flags();
+          auto saved_fill = output_.fill();
+          for (size_t i = 0; i < raw.size(); ++i) {
+            if (i > 0 && i % 16 == 0) output_ << " 0x";  // Add space every 16 bytes for readability
```

Comment:
Is this just a space? Why is there an extra 0x just in this case?

Reply:
Ok, perhaps a more basic question is: what format is this? Is it supported by the parser also? I don't see any changes to the parser to support this new format.